chef_master/source/release_notes_chefdk.rst

```diff
@@ -1,59 +1,173 @@
 =====================================================
-Release Notes: Chef Development Kit 0.19 - 3.10
+Release Notes: Chef Development Kit 0.19 - 4.0
 =====================================================
 `[edit on GitHub] <https://github.com/chef/chef-web-docs/blob/master/chef_master/source/release_notes_chefdk.rst>`__
 
 Chef Development Kit is released on a monthly schedule with new releases the third Monday of every month. Below are the major changes for each release. For a detailed list of changes, see the `ChefDK Changelog on GitHub <https://github.com/chef/chef-dk/blob/master/CHANGELOG.md>`__
 
+What's New in 4.0
+=====================================================
+
+* **Breaking Changes**
+
+  * **Chef EULA**
+
+    Usage of ChefDK 4.0, Chef Infra Client 15, and InSpec 4 requires accepting the `Chef EULA <https://docs.chef.io/chef_license.html#chef-eula>`__. See the `frequently asked questions <https://www.chef.io/bmc-faq/>`__ for information about the license update and the associated business model change.
+
+  * **Chef Provisioning**
+
+    Chef Provisioning is no longer included with Chef DK, and will be officially end of life on August 31, 2019. The source code of Chef Provisioning and the drivers have been moved into the chef-boneyard GitHub organization and will not be further maintained. Current users of Chef Provisioning should contact your Chef Customer Success Manager or Account Representative to review your options.
+
+  * ** knife bootstrap against cloud providers**
+
+    ``knife bootstrap`` was `rewritten <https://github.com/chef/chef/blob/cfbb01cb5648297835941679bc9638d3a823ad5e/RELEASE_NOTES.md#knife-bootstrap>`__ in Chef Infra Client 15. The ``knife-*`` cloud providers need to be updated to use this new API. As of ChefDK 4.0, ``knife bootstrap`` functionality against the cloud providers will be broken. We will fix this ASAP in a ChefDK 4.1 release. The only gem *not* affected is the ``knife-windows`` gem. It has already been re-written to leverage the new bootstrap library.
```

Comment:
@tas50 Are we wanting to use release notes github blobs for this? Or should we update the release note links?

Reply:
that just gets us a link to an exact commit so we make sure the links don't break later if we refactor the release notes in github.

Reply:
I could probably make it a friendly tag too like v15.0.300. That might be a better idea in the future